internal/compose/compose.go

```diff
@@ -67,7 +136,7 @@ func (p *Project) Down(opts CommandOptions) error {
 	args = append(args, "down")
 	args = append(args, opts.ExtraArgs...)
 
-	if err := runDockerComposeCmd(args, opts.Env); err != nil {
+	if err := p.runDockerComposeCmd(args, opts.Env, nil); err != nil {
```

Comment:
I'm admit I'm allergic to passing nil values as it tends to evolve into WinAPI "NULL" syndrome: `call(a, b, 1, 1, NULL, NULL, NULL, NULL, NULL, true, NULL, 1, NULL)` :)

I think you can repack all these args into `dockerComposeOptions`? Less refactoring in the future.

Reply:
Done in e0f21d7.